html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+    <style>
+      fieldset {
+        border: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html" method="post"></form>
+    <p>
+      <label for="email">Email</label>
+    </p>
+    <p>
+      <input type="email" id="email" name="user_email" />
+    </p>
+    <p>
+      <label for="coding-answer?">Why is coding cool?</label>
+    </p>
+    <select name="answer" id="answer-select">
+      <option value="">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+    <p>
+      <label for="favorite-language">Favorite Language (choose one)</label>
```

Comment:
Since this `label` isn't associated with any `input`, you should just put the text directly inside the `p` element